hack/merge-order.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -euo pipefail
+
+# Purpose
+# -------
+# Which PRs need to be backported to a branch that has been frozen for release purposes?
+#
+# This script will query Github to find all PRs that have merged into master and more recent release branches
+# since a release blocker issue was opened for the specified branch. The script will list the correct merge order of all
+# PRs that went in to make it easier to see what needs to be backported once the release blocker issue is closed.
+#
+# Install Required Tools
+# ----------------------
+# jq gh
+#
+# Usage
+# -----
+# * Ensure your local master branch is up-to-date
+# * Run this script from master:
+# * ./hack/merge-order.sh <branch_name>
+
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/util.sh
+
+function help() {
+  printf "${BRIGHT}"
+  echo "Usage: merge-order.sh branch_name"
+  echo "List merged PRs in order for all relevant branches since release blocker issue was opened for the given branch."
+  echo ""
+  echo "Example:"
+  echo "# Get merged PRs since code freeze for WMCO v8 for all branches above release-4.13 (master, release-4.14, etc.)"
+  echo "errata-check.sh release-4.13"
+  printf "${NORMAL}"
+}
+
+
+# Main script #
+if [ $# -eq 0 ]; then
+    print-error "Invalid arguments, no branch provided"
+    help
+    exit 1
+fi
+
+TARGET_BRANCH=$1
+BRANCH_VERSION=$(validate-target-branch "$TARGET_BRANCH")
+
+# Check for a blocking issue for the given branch
+rel_issue=$(gh issue list --state open --search "Merge blocker for active release | branch:$TARGET_BRANCH" \
```

Comment:
This may change, based on the conclusions of [How we manage our build pipeline](https://docs.google.com/document/d/1qE0Ym-syg2uOsEinH0mSujw0MYfUm_GBLZDqRFcTzic/edit)

Reply:
Yep, have an idea for this in https://github.com/openshift/windows-machine-config-operator/pull/1952#discussion_r1397550427